### PR TITLE
[API]Book-4-Implement Hotel EndPoints

### PR DESCRIPTION
**Pull request by Norhan605**

### CODE REVIEW

File: src/Services/BookingClone.Application/Features/HotelFeatures/Queries/GetById/GetByIdQueryHandler.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            return new GetHotelsDto {  Name = hotel.Name , Description=hotel.Description, Reviews = hotel.Reviews };
+            
+           
+
+        }
+        else
+        {
+            throw new Exception("Not Found A Hotel");
+        }
+    }
```
Comment (YoussefWaelMohamedLotfy) [inline src/Services/BookingClone.Application/Features/HotelFeatures/Queries/GetById/GetByIdQueryHandler.cs:44]: + Remove the Try-Catch Statement and return a null if not found
+ Use Automapper to map objects (See Reservation Handlers for reference)

File: src/Services/BookingClone.Application/Features/AttractionFeatures/EntityMappingConfig.cs [hunk 2]
```diff
@@ -12,5 +13,10 @@ public EntityMappingConfig()
         CreateMap<AttractionImage, AttractionImageDto>();
         CreateMap<AddAttractionDto, Attraction>();
         CreateMap<UpdateAttractionDto, Attraction>();
+        CreateMap<Hotel,GetHotelsDto>();
+        CreateMap<AddHotelsDto, Hotel>();
+        CreateMap<UpdateHotelDto, Hotel>();
```
[the remark this answers is not in the document]
Comment (Norhan605) [reply src/Services/BookingClone.Application/Features/AttractionFeatures/EntityMappingConfig.cs]: Done

File: src/Services/BookingClone.Application/Features/HotelFeatures/Queries/GetById/GetByIdQueryHandler.cs (hunk 1 above)
Comment (Norhan605) [reply src/Services/BookingClone.Application/Features/HotelFeatures/Queries/GetById/GetByIdQueryHandler.cs]: Done

**Review by YoussefWaelMohamedLotfy — APPROVED** [review]
Well done, Norhan, in completing the task 🎉

**Pull request merged**